governance/policies/README.md

```diff
@@ -0,0 +1,179 @@
+# opam Repository Policies
+
+This document lists the explicit policies governing the maintenance and curation
+of the opam repository.
+
+*NOTE*: This policy is not static and will be updated as needed.
+
+<a name="policy-0"></a>
+## 0. Packages must meet the criteria for inclusion
+
+Packages in this repository should meet the [criteria for inclusion](./archiving.md#inclusion-criteria)
+enumerated in the [archiving policy](./archiving.md#inclusion-criteria "ttp-equiv='Content-Type' content='text/html; charset=utf-8' /").
```

Comment:
```suggestion
enumerated in the [archiving policy](./archiving.md#inclusion-criteria).
```
was that correct?

Reply:
Sorry about that! Nov ene sure how I did that, but sorry for missing it in my pre-PR review.